mkdocs.yml

```diff
@@ -73,7 +73,6 @@ nav:
     - Claudie v0.3: CHANGELOG/changelog-0.3.x.md
   - How to work with docs:
```

Comment:
I think this whole section is kind of irrelevant for Claudie users. I would expect only Claudie devs to work with docs, not Claudie users. 

IMO, the docs.claudie.io is a user guide, not developer guide.